python_scripts/ensemble_bagging.py

```diff
@@ -127,6 +124,7 @@ def bootstrap_sample(data, target):
     data_bootstrap, target_bootstrap = bootstrap_sample(
         data_train,
         target_train,
+        seed=bootstrap_idx,
```

Comment:
I might add a small comment to mention that since we pass different seed, we ensure that the bootstrap samples are different but reproducible.